packages/webview_flutter/webview_flutter_wkwebview/lib/src/webview_cupertino_widget.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+
+import 'web_kit/web_kit.dart' as web_kit;
+
+/// A [Widget] that displays a [web_kit.WebView].
+class WebViewCupertinoWidget extends StatefulWidget {
```

Comment:
We should probably not call this Cupertino since we'll likely add macOS support to this package at some point, and in Flutter terminology `Cupertino` is specifically iOS. `WKWebViewWidget`? Although that could be confusing, especially if we put `WK` on the other classes. How about `WebViewWebKitWidget`?

Reply:
I used `Cupertino` because it is used as the name of the [WebViewPlatform implementation](https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/lib/src/webview_cupertino.dart#L18), but I think we should also update that with the new plugin interface. What are your thoughts on `WebKitWebViewWidget`? I was also, coincidently, updating the Android implementation to `AndroidWebKitWebViewWidget`.

This also made me consider that this plugin is called `webview_flutter_wkwebview`, but the Apple library is called [WebKit](https://developer.apple.com/documentation/webkit?language=objc) and [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc) is only a class in the library. However, the Android library could technically be called [WebKit](https://developer.android.com/reference/android/webkit/package-summary) too,  so I could understand why are plugin naming scheme wouldn't be consistent.

But all in all, I was considering

| Platform      | Platform Implementation Prefix |   Library File Name   |
| :---:        |    :----:   |          :---: |
| iOS/MacOS      | WebKit...       | web_kit.dart   |
| Android   | AndroidWebKit...        | android_web_kit.dart  |

Reply:
I like WKWebView because it's less ambiguous; webkit doesn't really say much if you don't know its intended to be a library name. UIWebView was also WebKit-based, for instance. While WKWebView is only one class, it's the core class, with all the others being supporting classes.